src/v/kafka/server/snc_quota_manager.cc

```diff
@@ -588,6 +659,13 @@ ingress_egress_state<quota_t> snc_quota_manager::get_quota() const noexcept {
     return {.in = _shard_quota.in.quota(), .eg = _shard_quota.eg.quota()};
 }
 
+ingress_egress_state<quota_t>
+snc_quota_manager::get_throughput() const noexcept {
+    return {
+      .in = _shard_quota.in.quota() - _shard_quota.in.get_current_rate(),
```

Comment:
a comment here would be useful. based on the function names, it looks like the current rate is the current throughput, so the subtraction isn't obvious.

Reply:
Actually @michael-redpanda has also nocticed that, so I'm adding a comment here